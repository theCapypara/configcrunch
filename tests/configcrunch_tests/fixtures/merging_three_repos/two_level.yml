base:
  $ref: /base_ref_level
  level_dict:
    base:
      $ref: /level_ref
      name: base
    three_base:
      $ref: /level_ref
      name: base
    two:
      base_ref:
        $ref: /base_ref