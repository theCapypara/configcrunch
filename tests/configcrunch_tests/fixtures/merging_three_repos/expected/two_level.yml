base:
  level_dict:
    base:
      $name: base
      name: base
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    one:
      $name: one
      name: one
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    two:
      $name: two
      name: two
      base_ref:
        more:
          one: one
          two: two
          three: three
          one_two: two
          two_three: three
          three_base: three
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    three:
      $name: three
      name: three
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    three_base:
      $name: three_base
      name: base
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    two_three:
      $name: two_three
      name: three
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three
    one_two:
      $name: one_two
      name: two
      more:
        one: one
        two: two
        three: three
        one_two: two
        two_three: three